File: token-validation-spring-demo/src/test/java/no/nav/security/token/support/demo/spring/LocalDemoApplication.java
package no.nav.security.token.support.demo.spring;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication
public class LocalDemoApplication {
	public static void main(String[] args) {
		SpringApplication app = new SpringApplication(LocalDemoApplication.class);
		app.setAdditionalProfiles("local");
		app.run(args);
	}
}

File: token-validation-spring-demo/src/test/kotlin/no/nav/security/token/support/demo/spring/LocalDemoApplication.kt
package no.nav.security.token.support.demo.spring

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
class LocalDemoApplication {

    fun main(args : Array<String>) {
        val app = SpringApplication(LocalDemoApplication::class.java)
        app.setAdditionalProfiles("local")
        app.run(*args)
    }
}